doc/dev_documentation/index.rst

```diff
@@ -0,0 +1,24 @@
+.. Deltares python Developer documentation documentation master file, created by
```

Comment:
I am not sure about this documentation structure. The whole point of all the docstrings is that we can generate the documentation form there, right?

Also, why do we have user-documentation in the repository? For which user user is that meant? If it are the design toolkit users then it should definitely not be documentation in this repo.